packages/passport/sdk/src/Passport.ts

```diff
@@ -137,15 +138,28 @@ export class Passport {
     return this.passportImxProviderFactory.getProvider();
   }
 
-  public connectEvm(): Provider {
-    return new ZkEvmProvider({
+  public connectEvm(options: {
+    announceProvider: boolean
```

Comment:
It gives more control to the developer around how they want to consume the provider.. they might have already integrated with Passport and didn't want to announce the provider again. Announcing the provider means it would potentially show up a list of injected providers within other widgets, wagmi injected connectors etc.